modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1145,6 +1145,16 @@
     <string name="settings_auto_download_filters_episodes">All filter episodes</string>
     <string name="settings_auto_download_new_episodes">New episodes</string>
     <string name="settings_auto_download_podcasts">Auto download podcasts</string>
+    <string name="settings_auto_download_new_episodes_description">Download new episodes as they’re released, and manage your storage by setting a limit on how many episodes are saved</string>
```

Comment:
```suggestion
    <string name="settings_auto_download_new_episodes_description">Download new episodes as they’re released, and manage your storage by setting a limit on how many episodes are saved.</string>
```
Other option descriptions have a full stop, so maybe we should match this.

![Screenshot_20241008_211053](https://github.com/user-attachments/assets/1d6295ac-be3f-419c-b4b9-5d7340e75657)